include/base/NekRSProblem.h

```diff
@@ -196,19 +208,22 @@ class NekRSProblem : public NekRSProblemBase
   /// displacement in z for all nodes from MOOSE, for moving mesh problems
   double * _displacement_z = nullptr;
 
+  /// mesh velocity for a given element, used internally for calculating mesh velocity over one element
+  double * _mesh_velocity_elem = nullptr;
+
   /// temperature transfer variable written to be nekRS
   unsigned int _temp_var;
 
   /// flux transfer variable read from by nekRS
   unsigned int _avg_flux_var;
 
-  /// x-displacment transfer variable read from for moving mesh problems
+  /// x-displacment transfer variable read from main app for moving mesh problems
```

Comment:
I don't think we need to have "main app" in this Doxygen comment - technically, the displacement could come from  a sub-app. This is super minor, but just want the comments to be as general as possible

Reply:
Thanks. How about
`/// x-displacment transfer variable` or `/// x-displacment transfer variable read for moving mesh problems`?

Reply:
I like the second one, but either is fine

Reply:
Done, thanks!